types/wordcloud/index.d.ts

```diff
@@ -10,14 +10,17 @@ declare function WordCloud(elements: HTMLElement | HTMLElement[], options: WordC
 
 declare namespace WordCloud {
     const isSupported: boolean;
-    let miniumFontSize: number;
+    const minFontSize: number;
+
+    /** Stop rendering. */
+    function stop(): void;
```

Comment:
I think this function only exists as of version 1.2, it doesn't seem to be present in version 1.1. If you'd still like to add this function, please update the version these typings correspond to in the header above, like this: `// Type definitions for wordcloud 1.2`